test/test_genome.py

```diff
@@ -660,6 +661,25 @@ def testNoFeaturesAtOffsetZero(self):
             },
             genome.offsetInfo(0))
 
+    def testLowCoverageGenome(self):
+        """
+        If the genome has insufficient coverage of the reference, None
```

Comment:
Shouldn't this say 'insufficient coverage of the genome'?

Reply:
No, it's correct. It's testing for insufficient coverage of the reference. I.e., that the genome sequence covers the reference sufficiently well.